From 3ab5c5e5a8695386d7eb029fe80906d0ce8ed0f3 Mon Sep 17 00:00:00 2001
From: Jaroslav Kysela <jkysela@redhat.com>
Date: Wed, 3 Sep 2014 07:35:19 -0400
Subject: [alsa] hda: fix a fixup value for codec alc293 in the pin_quirk table

Message-id: <140972823573-338-patchset-jkysela@redhat.com>
Patchwork-id: 91034
O-Subject: [RHEL 7.1 PATCH 337/442] ALSA: hda - fix a fixup value for codec alc293 in the pin_quirk table
Bugzilla: 1112200
RH-Acked-by: John Linville <linville@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Ivan Vecera <ivecera@redhat.com>

The fixup value for codec alc293 was set to
ALC269_FIXUP_DELL1_MIC_NO_PRESENCE by a mistake, if we don't fix it,
the Dock mic will be overwriten by the headset mic, this will make
the Dock mic can't work.

Cc: David Henningsson <david.henningsson@canonical.com>
Signed-off-by: Hui Wang <hui.wang@canonical.com>
Signed-off-by: Takashi Iwai <tiwai@suse.de>

(picked from kernel: 64eb428078b7b9b40c2d9413a7332fa3af85c51d)
rhbz#1112200
Signed-off-by: Jaroslav Kysela <jkysela@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/sound/pci/hda/patch_realtek.c b/sound/pci/hda/patch_realtek.c
index 9c095fa..06d6063 100644
--- a/sound/pci/hda/patch_realtek.c
+++ b/sound/pci/hda/patch_realtek.c
@@ -5159,7 +5159,7 @@ static const struct snd_hda_pin_quirk alc269_pin_fixup_tbl[] = {
    {0x1d, 0x40700001},
    {0x1e, 0x411111f0},
   },
-  .value = ALC269_FIXUP_DELL1_MIC_NO_PRESENCE,
+  .value = ALC293_FIXUP_DELL1_MIC_NO_PRESENCE,
  },
  {}
 };
-- 
1.7.1